### ISSUE TITLE
Notes?

### ISSUE THREAD

**Issue by Gobsmackaroonied** [luckydog7/Funkin-android#1808]
Have the notes been programmed in, or are we just listening to the songs at this point?

**Comment by HatsuneMiku-github** [comment]
Friday Night Funkin' for android uses the official source code for fnf so it isn't like those cringy RIP-offs and it's even the actual fnf but on android

**Comment by Gobsmackaroonied** [comment]
> Friday Night Funkin' for android uses the official source code for fnf so it isn't like those cringy RIP-offs and it's even the actual fnf but on android

Yes, I get that, but whenever I run it and try to play a song, there are no notes. The opposing sprite freezes whenever notes are supposed to be scrolling, but bf doesn't, and bf has all the "miss" animations, so I was wondering if this is intentional
